modules/configuring-the-training-operator-permissions-when-not-using-kueue.adoc

```diff
@@ -0,0 +1,156 @@
+
+.Procedure
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<admin_username>__ -p __<password>__
+----
+
+. Create a cluster role with the required access permissions, as follows:
+.. Create a YAML file named `pytorchjob_role.yaml`.
+.. Add the following `ClusterRole` object definition:
++
+[source]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pytorchjob-role
+rules:
+  - apiGroups:
+      - "kubeflow.org"
+    resources:
+      - pytorchjobs
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - "kubeflow.org"
+    resources:
+      - pytorchjobs/status
+    verbs:
+      - get
+----
+.. Save your changes in the `pytorchjob_role.yaml` file.
+.. Apply the configuration to create the `pytorchjob-role` object:
++
+[source]
+----
+$ oc apply -f pytorchjob_role.yaml
+----
+
+. Create a service account, as follows:
+.. Create a YAML file named `pytorchjob_serviceaccount.yaml`.
+.. Add the following `ServiceAccount` object definition:
++
+[source]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pytorchjob-sa
+  namespace: kfto
+----
+.. Replace the example namespace value `kfto` with the name of your project.
+.. Save your changes in the `pytorchjob_serviceaccount.yaml` file.
+.. Apply the configuration to create the `pytorchjob-sa` service account:
++
+[source]
+----
+$ oc apply -f pytorchjob_serviceaccount.yaml
+----
+
+. Assign the cluster role to the service account, as follows:
+.. Create a YAML file named `pytorchjob_rolebinding.yaml`.
+.. Add the following `ClusterRoleBinding` object definition:
++
+[source]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: pytorchjob-rolebinding
+subjects:
+- kind: ServiceAccount
+  name: pytorchjob-sa
+  namespace: kfto
+roleRef:
+  kind: ClusterRole
+  name: pytorchjob-role
+  apiGroup: rbac.authorization.k8s.io
+----
+.. Replace the example namespace value `kfto` with the name of your project.
+.. Save your changes in the `pytorchjob_rolebinding.yaml` file.
+.. Apply the configuration to assign the `pytorchjob-role` cluster role to the `pytorchjob-sa` service account:
++
+[source]
+----
+$ oc apply -f pytorchjob_rolebinding.yaml
+----
+
+
+.Verification
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, select your project from the *Project* list. 
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, select your project from the *Project* list.
+endif::[]
+. Click *Workloads -> Service Accounts* and verify that `pytorchjob-sa` is listed.
+. Click *Workloads -> Roles and Role Bindings* and verify that `pytorchjob-role` and `pytorchjob-rolebinding` are correctly created and assigned.
```

Comment:
ServiceAccounts and RoleBindings are available under `User management` menu, not `Workloads`

Reply:
Thank you, updated in follow-up PR 369.